components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -203,6 +204,11 @@ func hasSynced(ctx context.Context, fn waitForCacheSyncFunc) error {
 	return nil
 }
 
+// MsgChannelClient returns a clientset interface for messaging v1alpha1 API
+func (k *KnativeLib) MsgChannelClient() messagingv1alpha1Client.MessagingV1alpha1Interface {
```

Comment:
Why "Channel"? Either return a "Messaging" `MessagingV1alpha1Interface` or a "Channel" `ChannelsGetter` if you are only interested about Channels.

Reply:
Unfortunately, "messagingChannel" is misleading. I renamed it to "msgClient" as I'm interested in the clientset for messagingv1alpha1(github.com/knative/eventing/pkg/client/clientset/versioned/typed/messaging/v1alpha1) API.

Reply:
Sounds appropriate 👍